ContextDesktop/src/test/java/com/github/migi_1/Context/damageDealers/TestObstacleSpawner.java

```diff
@@ -38,18 +40,35 @@
     private Main main;
     private ServerWrapper serverWrapper;
     private Server server;
+    private MainEnvironment environment;
+    private Commander commander;
+    private BoundingBox leftBound;
+    private BoundingBox rightBound;
 
     /**
      * Setup for the tests.
+     * @throws Exception
+     * @throws SecurityException
+     * @throws NoSuchMethodException
      */
```

Comment:
FYI, I find this CheckStyle rule a bit too much in Test classes, so to bypass this CheckStyle error just add a space after each exception.